src/SmartComponents/RunTaskModal/ConversionTaskInputParameters.js

```diff
@@ -0,0 +1,197 @@
+import React, { useState } from 'react';
+import propTypes from 'prop-types';
+import {
+  Checkbox,
+  Form,
+  Icon,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons';
+
+const ConversionTaskInputParameters = ({
+  slug,
+  parameters,
+  setDefinedParameters,
+}) => {
+  const [elsDisabled, unavailableKmods, skipKernelCheck, skipPackageCheck] =
+    parameters;
+  const [elsDisabledValue, setElsDisabledValue] = useState(elsDisabled.default);
+  const [unavailableKmodsValue, setUnavailableKmodsValue] = useState(
+    unavailableKmods.default
+  );
+  const [skipKernelCheckValue, setSkipKernelCheckValue] = useState(
+    skipKernelCheck.default
+  );
+  const [skipPackageCheckValue, setSkipPackageCheckValue] = useState(
+    skipPackageCheck.default
+  );
+
+  const updateParameters = (parameter, newValue) => {
+    console.log(parameter.key, newValue);
+    setDefinedParameters((prevState) => {
+      return prevState.map((param) => {
+        if (param.key === parameter.key) {
+          return { key: param.key, value: newValue };
+        } else {
+          return param;
+        }
+      });
+    });
+  };
+  const toBool = (value) => {
+    return value.toLowerCase() === 'true' || value === '1';
```

Comment:
Can the value also be an int? So shouldn't we add something like
```suggestion
    return value.toLowerCase() === 'true' || value === '1' || value === 1;
```

Reply:
I'm not sure it can be an int, the parameters that are passed to us are all strings AFAIK, but a check for an int can't hurt.

Reply:
Done anyway.